native/spark-expr/src/cast.rs

```diff
@@ -138,240 +138,6 @@ pub struct Cast {
     pub cast_options: SparkCastOptions,
 }
 
-/// Determine if Comet supports a cast, taking options such as EvalMode and Timezone into account.
-pub fn cast_supported(
-    from_type: &DataType,
```

Comment:
I moved this into schema_adapter as a private function since it is specific to reading parquet

Reply:
+1